README.md

```diff
@@ -288,4 +288,14 @@ Using already existing Helm charts and operators
 - Converting bash scripts into ansible playbooks
 - Editing sudoers
 - Package module
-- What is yay
\ No newline at end of file
+- What is yay
+
+## 37 Vyos, instalation and configuration manual and via scripts, ansible playbook. Installing k3s cluster via ansible playbook
```

Comment:
37 is wrong, 38 is correct. And please leave only the part that was on the lecture (this concerns also README.md from the respective folder)